app/services/sfn_pipeline_dispatch_service.rb

```diff
@@ -9,6 +9,7 @@ class SfnPipelineDispatchService
   # Constains SFN deployment stage names that differ from Rails.env
   ENV_TO_DEPLOYMENT_STAGE_NAMES = {
     "development" => "dev",
+    "staging" => "staging",
```

Comment:
The idea here was to just use the environment name if key does not exist.
But it does not hurt.